### PR TITLE
[13.0][ADD] stock_picking_warn_message

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
### Stock Picking Warn Message

This module add a warning popup on pickings to ensure warning is populated
no only when partner is changed.

CC @ForgeFlow @MiquelRForgeFlow

### CODE REVIEW

File: stock_picking_warn_message/README.rst [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    :target: https://translation.odoo-community.org/projects/stock-logistics-workflow-13-0/stock-logistics-workflow-13-0-stock_picking_warn_message
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/154/13.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module add a warning popup on pickings to ensure warning is populated
+no only when partner is changed.
```
Comment (rousseldenis) [inline stock_picking_warn_message/README.rst:29]:
```suggestion
not only when partner is changed.
```

File: stock_picking_warn_message/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Stock Picking Warn Message",
+    "summary": """
+        Add a popup warning on picking to ensure warning is populated""",
+    "version": "13.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "ForgeFlow, Odoo Community Association (OCA)",
+    "website": "www.github.com/OCA/stock-logistics-workflow.git",
```
Comment (rousseldenis) [inline stock_picking_warn_message/__manifest__.py]:
```suggestion
    "website": "https://github.com/OCA/stock-logistics-workflow",
```

**Review by AdriaGForgeFlow — APPROVED** [review]
Functional review on local :+1:

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by HaraldPanten — APPROVED** [review]
Seems to be OK

**Review by AaronHForgeFlow — APPROVED** [review]

**Review by rousseldenis — CHANGES_REQUESTED** [review]
LGTM. Nitpicking

**Comment by HviorForgeFlow** [comment]
@rousseldenis comments attended

**Comment by HviorForgeFlow** [comment]
@rousseldenis can you move this forward?

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**